_dashboards/management/accelerate-external-data.md

```diff
@@ -14,35 +13,172 @@ Introduced 2.11
 
 ### Define covering index settings
 
-1. Under **Index settings**, enter a valid index name. Note that each Amazon S3 table can have multiple covering indexes.
-2. Once you have added the index name, define the covering index fields by selecting `(add fields here)` under **Covering index definition**. 
-3. Select the **Copy Query to Editor** button to apply your covering index settings.
-4. View the covering index query details in the table pane and then select the **Run** button. Your index is added to the left-side navigation menu containing the list of your databases.
+1. For **Index name**, enter a valid index name. Note that each table can have multiple covering indexes.
+2. Choose a **Refresh type**. By default, OpenSearch automatically refreshes the index. Otherwise, you must manually trigger a refresh using a REFRESH statement.
+3. Enter a **Checkpoint location**, which is a path for refresh job checkpoints. The location must be a path in an HDFS compatible file system. 
+4. Define the covering index fields by selecting **(add fields here)** under **Covering index definition**. 
+5. Select **Create acceleration** to apply your covering index settings.
+6. View the covering index query details and then click **Run**. OpenSearch adds your index to the left navigation pane.
+
+Alternately, you can manually create a covering index on your table using Query Workbench. Select your data source from the dropdown and run a query like the following:
+
+```sql
+CREATE INDEX vpc_covering_index
+ON datasourcename.gluedatabasename.vpclogstable (version, account_id, interface_id, 
+srcaddr, dstaddr, srcport, dstport, protocol, packets, 
+bytes, start, action, log_status STRING, 
+`aws-account-id`, `aws-service`, `aws-region`, year, 
+month, day, hour )
+WITH (
+  auto_refresh = true,
+  refresh_interval = '15 minute',
+  checkpoint_location = 's3://accountnum-vpcflow/AWSLogs/checkpoint'
+)
+```
+
+## Materialized views
+
+With _materialized views_, you can use complex queries, such as aggregations, to power Dashboards visualizations. Materialized views ingest a small amount of your data, depending on the query, into OpenSearch. OpenSearch then forms an index from the ingested data that you can use for visualizations. You can manage the materialized view index with Index State Management. For more information, see [Index State Management](https://opensearch.org/docs/latest/im-plugin/ism/index/).
+
+### Define materialized view settings
+
+1. For **Index name**, enter a valid index name. Note that each table can have multiple covering indexes.
+2. Choose a **Refresh type**. By default, OpenSearch automatically refreshes the index. Otherwise, you must manually trigger a refresh using a REFRESH statement.
```

Comment:
```suggestion
2. Choose a **Refresh type**. By default, OpenSearch automatically refreshes the index. Otherwise, you must manually trigger a refresh using a `REFRESH` statement.
```